### PR TITLE
Mp lxai

### PR DESCRIPTION
**Pull request by mpantoja314**
added the Program Committee Application Link
re-check that volunteer link is there 
and changed https://cmt3.research.microsoft.com/LXAINeurLPS2019 
to 
https://cmt3.research.microsoft.com/LXAINeurIPS2019

### CODE REVIEW

File: _includes/neurips/2019/about.neurips.2019 .es.html [hunk 1]
```diff
@@ -93,15 +93,18 @@ <h4>Agradecemos a los siguientes miembros de la comunidad LatinX en IA y a los a
...
+          <a class="project-link" href="https://forms.gle/orfBj15ehMyAgPyJA">Unirse al Comité</a>
       </div>
       <hr>
 	<div id="nipsVolunteer">
         	<hr>
         	<h3>Voluntarios</h3>
-        	<p>Si estas interesado en trabajar de voluntario en LXAI, por favor rellene el siguiente formulario: <a href="https://docs.google.com/forms/d/1RVx0LM_t0W0xz2cagq-DOb2sSlAjfsjHxITRgmaQwDI/edit?usp=sharing
-">Voluntarios</a></p>
-        	<hr>
+        	<p>Si estas interesado en trabajar de voluntario en LXAI, por favor rellene el siguiente formulario:
```
Comment (juancamilog) [inline _includes/neurips/2019/about.neurips.2019 .es.html]: "Si te interesa participar como voluntario durante el evento de LXAI, por favor llena el siguiente formulario"

File: _includes/neurips/2019/about.neurips.2019 .es.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h2>Instrucciones de envio</h2>
...
+        <p>Todos los articulos enviados seran sometidos a una revision doble sin nombre de autor. Las presentaciones pueden tener hasta dos páginas, incluidas todas las figuras y tablas, con una página adicional para las referencias enviadas como un enlace a un PDF, siguiendo el <a href="https://nips.cc/Conferences/2019/PaperInformation/StyleFiles"> estilo estandard </a> de la conferencia. Las presentaciones deben estar en una sola columna. Deben estar compuestas por tipos de letra de 11 puntos y deben tener al menos un margen de 1 pulgada alrededor. Las presentaciones que no sigan estas pautas corren el riesgo de ser rechazadas sin consideración de sus méritos. Las presentaciones deben indicar el problema de investigación, la motivación y la contribución técnica. Las presentaciones deben ser independientes e incluir todas las figuras, tablas y referencias.</p>
```
Comment (juancamilog) [inline _includes/neurips/2019/about.neurips.2019 .es.html]: "Todos los artículos envíados serán sometidos a una evaluación de doble ciego (autores y evaluadores anónimos)"

File: _includes/neurips/2019/about.neurips.2019.html [hunk 3]
```diff
@@ -40,7 +40,7 @@ <h2>Submission Instructions</h2>
...
+        <p>All submissions will be double-blind (they should not contain author names) and may be up to two pages including all figures and tables, with an additional page for references. The papers must be submitted as a PDF created with the <a href="https://nips.cc/Conferences/2019/PaperInformation/StyleFiles"> NeurIPS style format </a>. All submissions must be in English. Submissions that do not follow these guidelines risk being rejected without consideration of their merits. Submissions must state the research problem, motivation, and technical contribution including experimental results when applicable. Submissions must be self-contained and include all figures, tables, and references. The submission deadline is 23:59 September 13th, 2019 AoE. Please note that no extensions will be offered for submissions.
```
Comment (juancamilog) [inline _includes/neurips/2019/about.neurips.2019.html]: The papers must be submitted as a PDF created by following the <a href="https://nips.cc/Conferences/2019/PaperInformation/StyleFiles">NeurIPS style guide</a> (template also available in <a href="https://www.overleaf.com/latex/templates/neurips-2019/tprktwxmqmgk">Overleaf</a>).

**Comment by mpantoja314** [comment]
did all the above changes plus added the travel grant link. Resolved some conflicts 

It can be merged with master once double checked of course

**Comment by quickresolve** [comment]
Closing this pull request. Please submit changes requested to Spanish pages separately.

**Pull request closed without merging**